pkg/operator/ceph/object/objectstore.go

```diff
@@ -700,51 +700,71 @@ func CreatePools(context *Context, clusterSpec *cephv1.ClusterSpec, metadataPool
 }
 
 func createSimilarPools(ctx *Context, pools []string, clusterSpec *cephv1.ClusterSpec, poolSpec cephv1.PoolSpec, pgCount, ecProfileName string) error {
-	waitGroup, _ := errgroup.WithContext(context.TODO())
+	opPodResources := false
```

Comment:
How about this to describe the intention? Then set it to false when the env var is found
```suggestion
	configurePoolsConcurrently := true
```

Reply:
Done.